docs/core-concepts/raw-queries.md

```diff
@@ -20,7 +20,7 @@ const users = await sequelize.query("SELECT * FROM `users`", { type: QueryTypes.
 // We didn't need to destructure the result here - the results were returned directly
 ```
 
-Several other query types are available. [Peek into the source for details](https://github.com/sequelize/sequelize/blob/main/src/query-types.ts).
+Several other query types are available. [Peek into the source for details](https://github.com/sequelize/sequelize/blob/v6/src/query-types.js).
```

Comment:
```suggestion
Several other query types are available. [Peek into the source for details](https://github.com/sequelize/sequelize/blob/main/packages/core/src/query-types.ts).
```
These are not the versioned docs so they should direct to v7 code. A simple update of the new location should be fine, I don't expect them to move again soon

Reply:
Specifically this is the docs for v7 which is main, the doc for V6 is in the versionned_docs/V6 folder

Reply:
rip!